test/jobs/create_gemini_recipient_ids_job_test.rb

```diff
@@ -68,5 +68,21 @@ class CreateGeminiRecipientIdsJobTest < ActiveJob::TestCase
         recipient_id: recipient_id_channel
       )
     end
+
+    it "removes the updates the recipient ID status if there's a duplicate recipient_id" do
```

Comment:
Reading the test is clear, but the description string here is confusing.

Reply:
I can't say I disagree, but I don't think it's a blocker.

Reply:
I'll fix it in my follow ups.  I'll have to rebase anyway.